### DISCUSSION TITLE
Gnss/ins files' format

### DISCUSSION THREAD

**Discussion by ingcristianamaya** [rodralez/NaveGo#48] [Q&A] [answered]
I would like to know what are the formats used in this 2 files for put my files

**Comment by rodralez** [comment, marked as answer]
Hello,

I do not understand your question very well but I can say that the format of the input data to NaveGo are data structures in MATLAB standard format (.mat files). In the synthetic example file you will see a detailed explanation of these data structures.

Kind regards.

**Reply by ingcristianamaya** [reply]
Ok. Thanks but, is it possible to use a rinex and Imu data like inputs for NaveGo?

**Reply by rodralez** [reply]
For RINEX files, you can:

1) Process RINEX files with the rtklib software.
2) Generate a .pos file.
3) Process the .pos file with function rtkpos_read() from the data-acquisition directory.

For IMU data, I have develop the microstrain_read() and xsens_read() for Microstrain and Xsens IMUs, respectively. But you may modified these files according to your needs. 

Kind regards.
